etna/core/models/settings.py

```diff
@@ -12,15 +13,23 @@
 
 @register_setting(icon="list-ul")
 class SiteSettings(BaseSetting, ClusterableModel):
-    panels = [
-        InlinePanel(
-            "main_menu_items_rel",
-            heading="main menu items",
-            label="item",
-            min_num=1,
-            max_num=6,
-        ),
-    ]
+    stand_first = models.CharField(max_length=200, blank=False, null=True)
+    banner_link = models.URLField(blank=True, null=True, help_text="Banner URL")
```

Comment:
We should lose the `null=True` here, and again look to make the field name less ambiguous. I think `beta_banner_link` would be good. I would question the usefulness of the `help_text` here too. Is that supposed to be `verbose_name` instead of `help_text`?